src/Mutator/Unwrap/AbstractUnwrapMutator.php

```diff
@@ -35,6 +35,8 @@
 
 namespace Infection\Mutator\Unwrap;
 
+use function array_key_exists;
+use const Generator;
```

Comment:
https://travis-ci.org/infection/infection/jobs/617500222#L358

> PHP Fatal error:  Declaration of Infection\Mutator\Unwrap\UnwrapArrayChangeKeyCase::getParameterIndexes(PhpParser\Node\Expr\FuncCall $node): Generator must be compatible with Infection\Mutator\Unwrap\AbstractUnwrapMutator::getParameterIndexes(PhpParser\Node\Expr\FuncCall $node): Infection\Mutator\Unwrap\Generator in /home/travis/build/infection/infection/src/Mutator/Unwrap/UnwrapArrayChangeKeyCase.php on line 55

Is it a bug in PHP-CS-Fixer?

Reply:
Yes, but that we can manually fix not an issue

Reply:
I mean correct it manually and at the next run it won't cause any issue; the issue there was likely a wrong guess to convert `Generator` to an import for some reasons